x-pack/plugins/apm/server/routes/source_maps/route.ts

```diff
@@ -91,7 +114,7 @@ const uploadSourceMapRoute = createApmServerRoute({
   endpoint: 'POST /api/apm/sourcemaps 2023-10-31',
   options: {
     tags: ['access:apm', 'access:apm_write'],
-    body: { accepts: ['multipart/form-data'] },
+    body: { accepts: ['multipart/form-data'], maxBytes: 100 * 1024 * 1024 },
```

Comment:
Does it override the option https://www.elastic.co/guide/en/kibana/current/settings.html#server-maxPayload?

Reply:
According to the core team, this option sets the limit at the route level but there is no option for the user to increase the value. We need to introduce a new config setting for the endpoint.

I think we can proceed with the current approach and handle it in a follow-up work if is needed